box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxSearchFragment.java

```diff
@@ -166,20 +167,6 @@ private void onItemsFetched(BoxResponseIntent intent) {
         mSwipeRefresh.setRefreshing(false);
     }
 
-    @Override
-    protected void onBindBoxItemViewHolder(BoxItemViewHolder holder) {
-        if (holder.getItem() == null || holder.getItem().getBoxItem() == null) {
-            return;
-        }
-        final BoxItem item = holder.getItem().getBoxItem();
-        holder.getNameView().setText(item.getName());
-        holder.getMetaDescription().setText(BoxSearchListAdapter.createPath(item, File.separator));
```

Comment:
Fixed. I forgot i wanted this overriden in the viewholder instead of the fragment